fc-community-backend/apps/api/scripts/finaliseNotes.mjs

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env node
 
-import { FactChainBackend } from "../dist/factchain-core/web3.js";
+import { BlockchainConnector } from "../dist/factchain-core/web3.js";
 import { NoteService } from "../dist/factchain-core/noteService.js";
 import { config } from "../dist/factchain-core/env.js";
 import { getNetworkConfig } from "../dist/factchain-core/networks/config.js"
 
-const LOOKBACK_DAYS = parseInt(config.LOOKBACK_DAYS);
+const LOOKBACK_DAYS = 20;
```

Comment:
I want to avoid any confusion with the *365* default value from the Config since the finalization continues to rely on the `BlockchainConnector` to perform write actions.